core/src/main/java/com/exadel/etoolbox/linkinspector/core/services/data/impl/ConfigResourceListenerImpl.java

```diff
@@ -100,7 +100,7 @@ private Configuration getConfiguration(String pid) {
                 log.warn("Configuration for pid {} is not found", pid);
             }
         } catch (Exception e) {
-            log.error("Failed to get configuration for pid {}", pid, e);
+            log.info("Failed to get configuration for pid {}", pid, e);
```

Comment:
I believe that errors in this class are legit errors, they can tell if user settings manipulations are broken in production. This is not the same thing as an exception testing a link. Please restore the errorlevel.

Reply:
done